kogito-usertasks-with-security-oidc-keycloak/README.md

```diff
@@ -0,0 +1,211 @@
+# Kogito user task orchestration
+
+## Description
+
+A quickstart project shows very typical user task orchestration. It comes with two tasks assigned
+to human actors via groups assignments - `managers`. So essentially anyone who is a member of that 
+group can act on the tasks. Though this example applies four eye principle which essentially means 
+that user who approved first task cannot approve second one. So there must be always at least two 
+distinct manager involved.
+
+This example shows
+
+* working with user tasks
+* four eye principle with user tasks
+	
+	
+<p align="center"><img width=75% height=50% src="docs/images/process.png"></p>
+
+
+## Build and run
+
+### Prerequisites
+ 
+You will need:
+  - Java 1.8.0+ installed 
+  - Environment variable JAVA_HOME set accordingly
+  - Maven 3.5.4+ installed
+  - jq tool
+  - Docker
+
+
+When using native image compilation, you will also need: 
+  - GraalVM 19.1+ installed
+  - Environment variable GRAALVM_HOME set accordingly
+  - Note that GraalVM native image compilation typically requires other packages (glibc-devel, zlib-devel and gcc) to be installed too, please refer to GraalVM installation documentation for more details.
+
+###Starting and Configuring the Keycloak Server
+
+To start a Keycloak Server you can use Docker and just run the following command:
+
+```
+docker run --name keycloak -e KEYCLOAK_USER=admin -e KEYCLOAK_PASSWORD=admin -p 8280:8080 quay.io/keycloak/keycloak
+```
+
+You should be able to access your Keycloak Server at [localhost:8280/auth](http://localhost:8280).
+
+Log in as the admin user to access the Keycloak Administration Console. Username should be admin and password admin.
+
+Import the realm configuration [file](config/quarkus-realm.json) to create a new realm. For more details, see the Keycloak documentation about how to 
```

Comment:
importing of the config file fails due to duplicated roles and then it fails with messages:

`Error! Realm role 'uma_authorization' already exists.`

if I select that the import should override existing then I can this

`Error! App doesn't exist in role definitions: realm-management`

and if I skip the import of client roles then I cannot login anymore. Could you provide a bit of help on this? Maybe we import too much....

Reply:
@mswiderski yes, I see what is happening.. you have to import the file after clicking 'Add Realm' button. this errors are thrown because you are trying to import the 'quarkus realm' on 'Master realm'.. Here you need to create a new one and import there the file